docs/dictionary/control_st/repeat.lcdoc

```diff
@@ -6,20 +6,24 @@ Syntax:     repeat <loopForm>
       <statementList>
     end repeat
     
-Syntax:  repeat {forever | while <condition> | until <condition>} 
+Syntax:     repeat {forever | while <condition> | until <condition>} 
       <statementList> ; end repeat
```

Comment:
There's still a semicolon here. Also you can delete the 
```
repeat <loopForm>
    <statementList>
end repeat
```
as it's now covered by all the other forms, and delete the `loopForm` parameter description